tests/foreman/endtoend/test_cli_endtoend.py

```diff
@@ -54,339 +54,327 @@
-            5. Create a custom YUM repository
-            6. Create a custom PUPPET repository
-            7. Enable a Red Hat repository
-            8. Synchronize the three repositories
-            9. Create a new content view
-            10. Associate the YUM and Red Hat repositories to new content view
-            11. Add a PUPPET module to new content view
-            12. Publish content view
-            13. Promote content view to the lifecycle environment
-            14. Create a new activation key
-            15. Add the products to the activation key
-            16. Create a new libvirt compute resource
-            17. Create a new subnet
-            18. Create a new domain
-            19. Create a new hostgroup and associate previous entities to it
-            20. Provision a client
-
-        :id: 8c8b3ffa-0d54-436b-8eeb-1a3542e100a8
-
-        :expectedresults: All tests should succeed and Content should be
-            successfully fetched by client.
-        """
-        # step 1: Create a new user with admin permissions
-        password = gen_alphanumeric()
-        user = make_user({'admin': 'true', 'password': password})
-        user['password'] = password
-
-        # step 2.1: Create a new organization
-        org = self._create(user, Org, {'name': gen_alphanumeric()})
-
-        # step 2.2: Clone and upload manifest
-        if self.fake_manifest_is_set:
-            with manifests.clone() as manifest:
-                ssh.upload_file(manifest.content, manifest.filename)
-            Subscription.upload({'file': manifest.filename, 'organization-id': org['id']})
-
-        # step 2.3: Create a new lifecycle environment
-        lifecycle_environment = self._create(
-            user,
-            LifecycleEnvironment,
-            {'name': gen_alphanumeric(), 'organization-id': org['id'], 'prior': 'Library'},
-        )
 
-        # step 2.4: Create a custom product
-        product = self._create(
-            user, Product, {'name': gen_alphanumeric(), 'organization-id': org['id']}
-        )
-        repositories = []
 
-        # step 2.5: Create custom YUM repository
-        yum_repo = self._create(
-            user,
-            Repository,
+@pytest.fixture(scope='module')
+def fake_manifest_is_set():
+    return setting_is_set('fake_manifest')
+
+
+@pytest.mark.tier1
+@pytest.mark.upgrade
+def test_positive_cli_find_default_org():
+    """Check if 'Default Organization' is present
+
+    :id: 95ffeb7a-134e-4273-bccc-fe8a3a336b2a
+
+    :expectedresults: 'Default Organization' is found
+    """
+    result = Org.info({'name': DEFAULT_ORG})
+    assert result['name'] == DEFAULT_ORG
+
+
+@pytest.mark.tier1
+@pytest.mark.upgrade
+def test_positive_cli_find_default_loc():
+    """Check if 'Default Location' is present
+
+    :id: 11cf0d06-78ff-47e8-9d50-407a2ea31988
+
+    :expectedresults: 'Default Location' is found
+    """
+    result = Location.info({'name': DEFAULT_LOC})
```

Comment:
In this PR you're just dropping the reliance on pytest, so I will not consider this a required change now, but there is a problem with this test pattern.

The first two test functions are making the assumption that some org/location has been created without any fixture association or use, and I think they provide very little value to the test framework.  Their docblock describes behavior that should be done in the fixture that created the default or or default location in the first place.

In principal what should be tested by the two test names is valuable test coverage, but has been defined ineffectively.

I'd like to reconsider what's captured in the 'end_to_end' tests as a whole, because I think we already have this test coverage in spades.  We can build our pytest framework hooks and fixture structures to be more reliable with existence of session scoped test resources, and not use test cases to provide this mechanism.